go.mod

```diff
@@ -3,33 +3,36 @@ module github.com/openshift/assisted-service
 go 1.13
 
 require (
+	cloud.google.com/go/bigquery v1.8.0 // indirect
```

Comment:
i am really not sure why we have all this changes in go.mod. I don't see that u add new packages or i have missed it?
If no please drop those changes

Reply:
done.